p2panda-rs/src/schema/schema.rs

```diff
@@ -60,6 +61,19 @@ impl Schema {
     pub fn as_cddl(&self) -> String {
         generate_cddl_definition(&self.fields)
     }
+
+    /// Returns a unique string identifier for this schema
+    ///
+    /// This has the format "<schema name>_<hashed schema document view graph tips>".
+    pub fn hash_id(&self) -> String {
+        format!("{}_{}", self.name, self.id.hash().as_str())
+    }
+}
+
+impl Display for Schema {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", self.hash_id())
+    }
 }
```

Comment:
Carrying on from the comment above, maybe this is the place where I would expect the hash string to be the commonly used id. Hmmm.... basically I still need a little more thought on this. Would be happy to hear yours!